client/src/components/product/productCard.tsx

```
@@ -15,23 +17,46 @@
+  const [averageRating, setAverageRating] = useState(0);
+  const [totalReviews, setTotalReviews] = useState(0);
+
+  // Fetch product rating from the backend
+  useEffect(() => {
+    const fetchProductRating = async () => {
+      try {
+        const { data } = await axios.get(
+          `${process.env.NEXT_PUBLIC_BASE_URL}/api/review/rating/${product._id}`
+        );
+
+        if (data && data.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)